jsonschema2pojo-cli/src/main/java/org/jsonschema2pojo/cli/Arguments.java

```diff
@@ -418,4 +421,9 @@ public boolean isFormatDateTimes() {
         return formatDateTimes;
     }
 
+    @Override
+    public String isCustomSeparatorCharacters() {
```

Comment:
There are a few getters in this and other classes that have an incorrect `is` prefix instead of `get`. This and others should be `get`.